pkg/executor/sample.go

```diff
@@ -282,7 +282,19 @@ func (s *tableRegionSampler) buildSampleColAndDecodeColMap() ([]*table.Column, m
 		}
 	}
 	// Schema columns contain _tidb_rowid, append extra handle column info.
-	if len(cols) < len(schemaCols) && schemaCols[len(schemaCols)-1].ID == model.ExtraHandleID {
+	hasExtraHandleID := false
```

Comment:
Do we need  to add a simple test to make sure it will not break select ... tablesample ...?

Reply:
There are some existing test cases for the table sample feature, and they don't fail. I added a simple test case to verify `select _tidb_commit_ts from ... tablesample ...`.